docs/UserGuide.md

```diff
@@ -115,15 +114,15 @@ for the team of AddressBook-Level 3.
 
 - Please refer to this user guide whenever you feel something is wrong with the application to determine whether it is a genuine problem.
 - If you are a more technical or experienced user, feel free to check out our [Developer Guide](DeveloperGuide.md) for detailed technical information about TrueRental.
-- The **client** referred to in this user guide is the property owner or landlord who has engaged you to rent out and manage their property.
+- The **client** referred to in this user guide is the property owner or landlord who has engaged you to manage their property.
 - The **rental information** referred to in this user guide consists of the details of the property being rented, including key information for example address, rental start date, and other relevant property details.
-- The **customer** (as part of the rental information) referred to in this user guide is the tenant of the property managed by the client.
+- The **customer** (as part of the rental information) referred to in this user guide is the tenant of the property owned by the client.
 - **Monthly rent** is equivalent to **monthly rent amount**.
   - Both are referring to the monthly rent amount for the property managed by your client.
 - **Deposit** is equivalent to **deposit amount**.
-  - Both are referring to the security deposit amount for the property managed by your client, paid by the tenant at the start of the lease.
+  - Both are referring to the security deposit amount for the property owned by your client, paid by the tenant at the start of the lease.
 - **Customer list** is equivalent to **customers**.
-  - Both are referring to the name(s) of the tenant(s) for the property managed by your client.
+  - Both are referring to the name(s) of the tenant(s) for the property owned by your client.
```

Comment:
Client may or may not "owned" the property, see what are others' views on this.

Reply:
Owned is fine, as security deposit are usually given to landlords/owners.